packages/scrollable-image/src/components/scroll-horizontal.js

```diff
@@ -88,45 +88,36 @@ class ScrollHorizontal extends React.PureComponent {
       this.isDistanceFromTopSet = true
     }
 
+    if (window.innerHeight !== this.lastWindowHeight) {
+      this.handleResize()
+    }
```

Comment:
Since `handleResize` function works well, do we need this condition here?
Could we remove `this.lastWindowHeight`?